src/Nox.Types.EntityFramework/Types/User/UserConverter.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using System.Linq.Expressions;
+
+namespace Nox.Types.EntityFramework.Types;
+
+/// <summary>
+/// The user converter.
+/// </summary>
+public class UserConverter : ValueObjectConverter<User, string>
```

Comment:
Since ValueObjectConverter has been deleted we need to change the implementation of this converter to inherit from EFs ValueConverter.